### ISSUE TITLE
Code cleanup change `private protected` to `protected`

### ISSUE THREAD

**Issue by heku** [codecadwallader/codemaid#860]
### Environment
- Visual Studio version: VS 2019 and VS 2022 
- CodeMaid version: 12.0
- Code language: C#

### Description
Code cleanup change [`private protected`](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/private-protected) to `protected`

### Steps to recreate
```csharp
internal class Class1
{
    private protected Class1()
    {
    }
}
```
After run **Cleanup**, the `private protected` access modifier is updated to `protected` mistakenly.
```csharp
internal class Class1
{
    protected Class1()
    {
    }
}
```

**Comment by codecadwallader** [comment]
Thanks for reporting the issue.  I am able to reproduce it as well.  It doesn't look like the API we utilize recognizes that access modifier.

https://github.com/dotnet/roslyn/issues/22406

If there was a mapping we would want to extend this list: https://github.com/codecadwallader/codemaid/blob/dev/CodeMaidShared/Helpers/CodeElementHelper.cs#L65

**Comment by heku** [comment]
Yes, I thought it's a regex issue, but after debugging I found it's the emum doesnt have that value. I have no.idea to fix it, so posted it here.

**Comment by hagarpoa** [comment]
I'm also experiencing the same problem.

Environment
Visual Studio version: VS 2019
CodeMaid version: 12.0
Code language: C#
Description
Code cleanup change private protected to protected

**Comment by RichardMelito** [comment]
I can confirm that this is still a problem.

**Comment by OscarGil-Sage** [comment]
Same here. The issue persists.

**Comment by AndreiDeacUiPath** [comment]
This is still an issue , I eneded up disabling tclean up on save when deailing with certain files or going and reverting each cange by hand. Is really frustrating because it breaks the build.

**Comment by Banane9** [comment]
> dotnet/roslyn#22406

Just ran into this - they apparently closed the issue and won't implement it :C
